config/environments/production.rb

```diff
@@ -78,4 +78,6 @@
   if ENV["RAILS_LOG_TO_STDOUT"].present?
     config.logger = ActiveSupport::Logger.new(STDOUT)
   end
+
+  config.ckan_redirection_url = ENV['CKAN_REDIRECTION_URL'].present?
```

Comment:
This will only create the config variable in production.  It also needs to be specified in `config/environments/staging.rb`.

Additionally, `.present?` returns true or false, rather than the actual value.

Reply:
Might it be better to just define the URL here rather than needing to use an environment variable?

Reply:
@brucebolt I'm removing the `.present?`, but I'm not sure about putting it in `staging.rb` because that file includes production: `require_relative './production'`
doesn't this mean that it will inherit all the values defined in production? There is a similar thing we do for Django config so I assumed this was the equivalent, but I may be wrong

About the env variable: I usually use them so whenever you need to fix a value there is no need to change the source code, you will just change the value in the environment. But if it's not something you normally do, no problem, I can use the fix value